src/server/actions/study.actions.ts

```diff
@@ -189,14 +189,15 @@ export const approveStudyProposalAction = memberAction(async (studyId: string) =
 
         let status: StudyJobStatus = 'CODE-APPROVED'
 
-        if (USING_S3_STORAGE) {
+        // if we're not using s3 then containers will never build so just mark it ready
```

Comment:
not really s3 related, this could be:
```suggestion
        // if we're not connected to AWS codebuild,  containers will never build so just mark it ready
```